Pod/Classes/SearchableProtocol.swift

```diff
@@ -31,6 +31,7 @@ public protocol Searchable {
     func configureActivity(searchActivity: NSUserActivity, infoProvider: SearchInfoProvider) -> NSUserActivity
 }
 
+#if !os(tvOS)
 @available(iOS 9.0, *)
```

Comment:
`@available` in swift has a space for platform I think you can get that in there in the platform name spot - in this case, list non tvos platforms